### PR TITLE
OSDOCS-9443: Update note to reflect that US GovCloud is supported

### PR DESCRIPTION
**Pull request by bmcelvee**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-9443

Link to docs preview:
https://70835--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-service-definition#rosa-sdpolicy-regions-az_rosa-service-definition

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/rosa-sdpolicy-am-regions-az.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ endif::rosa-with-hcp[]
 
 [NOTE]
 ====
-China and GovCloud (US) regions are not supported, regardless of their support on OpenShift 4.
+China regions are not supported, regardless of their support on OpenShift 4.
 ====
 
 .AWS Regions
```
Comment (chadcrum) [inline modules/rosa-sdpolicy-am-regions-az.adoc:28]: The two GovCloud regions should be added to the list:
```
us-gov-west-1 - (AWS GovCloud - US-West)	
us-gov-east-1 - (AWS GovCloud - US-East)
```

File: modules/rosa-sdpolicy-am-regions-az.adoc [hunk 2]
```diff
@@ -22,7 +22,7 @@ endif::rosa-with-hcp[]
 
 [NOTE]
 ====
-China and GovCloud (US) regions are not supported, regardless of their support on OpenShift 4.
```
Comment (chadcrum) [inline modules/rosa-sdpolicy-am-regions-az.adoc:25]: Can you add a note that for GovCloud regions the customer will need to visit this page https://console.redhat.com/openshift/create/rosa/govcloud to request access?

**Comment by bmcelvee** [comment]
Thanks, @chadcrum! Updates are in and the build may take a few minutes to refresh

**Review by JoeAldinger — COMMENTED** [review]
/lgtm

**Pull request merged**